securedrop_client/logic.py

```diff
@@ -347,6 +336,8 @@ def setup(self):
         self.export.moveToThread(self.export_thread)
         self.export_thread.start()
 
+        storage.clear_download_errors(self.session)
```

Comment:
Confirming: This makes it so that the client has to be fully restarted  in order to clear the download errors and retry ((ogging out and logging back in again will not reset the error state).

Reply:
Right.